### DISCUSSION TITLE
Best way to reference a company wide shared dictionary file, multiple repos

### DISCUSSION THREAD

**Discussion by jamesmilanhiggins** [streetsidesoftware/vscode-spell-checker#806] [Q&A]
Hi.
I am wondering what the best way to maintain and use a company wide shared dictionary, across multiple repos. We have a team with multiple devs that all work on multiple repos, and we would like to have 1 common/shared dictionary file that we can maintain that we all reference with our spell check plugins.

**Comment by Jason3S** [comment]
This is a good question.

## Using a company repository

Example dictionary package: [cspell-dicts medical terms](https://github.com/streetsidesoftware/cspell-dicts/tree/master/dictionaries/medicalterms). Note, you do not need to compress the dictionary or use the `cspell-tools` to compile the dictionary, that is only needed for a large dictionary. A dictionary file is just a text file with 1 term per line. It is case sensitive. It does not need to be sorted.

Here is an example `cspell.json` file that the `cspell` cli uses to load dictionaries: [cspell/cspell-default.json](https://github.com/streetsidesoftware/cspell/blob/master/packages/cspell-bundled-dicts/cspell-default.json)

This approach works well if you are using `cspell` as part of your CI/CD process.
